groovy/groovy.editor/src/org/netbeans/modules/groovy/editor/api/lexer/LexUtilities.java

```diff
@@ -446,6 +511,15 @@ public static boolean isBeginToken(TokenId id, BaseDocument doc, int offset) {
         return END_PAIRS.contains(id);
     }
 
+    /**
+     * Return true iff the given token is a token that should be matched
```

Comment:
Possible typo here 'iff' . If it's really typo, it appears in more places.

Reply:
'iff' usually means 'if and only if'.